playground/dynamic-import-retry/vue-router/src/vite-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="vite/client" />
```

Comment:
I believe we can remove this file and reference the types in the tsConfig

Reply:
Updated